fastlane/Appfile

```diff
@@ -0,0 +1,5 @@
+app_identifier("org.covidwatch.tags.ios") # The bundle identifier of your app
+apple_id 'jeffctown@gmail.com'
```

Comment:
Just confirming: this will change once you figure out which account to use for TestFlight?

Reply:
Ooh, I should've explained this. Thanks for calling it out.  

This probably looks like it's uploading to my personal apple account, but it isn't.  Zsombor added me to his apple developer account where we already have a Tags build approved.  We want to keep using his account for now.